client/src/components/FoodSeeker/SearchResults/ResultsMap/ResultsMap.js

```diff
@@ -183,7 +188,15 @@ const ResultsMap = (
       )}
       <Map.NavigationControl
         showCompass={false}
-        className={classes.navigationControl}
+        style={{ top: 8, right: 8}}
+      />
+      <Map.ScaleControl
```

Comment:
Since ScaleControl was removed in #1882, should this be added back as part of styling update?

Reply:
The problem has been successfully addressed. A huge thank you to @SAUMILDHANKAR for your invaluable support and collaborative efforts in resolving the development issue.

Reply:
This looks resolved. Thanks for fixing this.